client/export.go

```diff
@@ -0,0 +1,312 @@
+	switch format {
+	case "json":
+		data, err := json.MarshalIndent(allEvents, "", "  ")
+		if err != nil {
+			return fmt.Errorf("failed to marshal JSON: %w", err)
+		}
+		buf.Write(data)
+
+	case "yaml":
+		data, err := yaml.Marshal(allEvents)
+		if err != nil {
+			return fmt.Errorf("failed to marshal YAML: %w", err)
+		}
+		buf.Write(data)
+
+	case "csv":
+		if err := writeCSV(buf, allEvents, defaultListKeyOrder); err != nil {
+			return fmt.Errorf("failed to write CSV: %w", err)
+		}
+
+	default:
+		return fmt.Errorf("unsupported format: %s", format)
+	}
+
+	return nil
+}
+
+// ExportCmd represents the export command
+var ExportCmd = &cobra.Command{
+	Use:   "export",
+	Args:  cobra.ExactArgs(0),
+	Short: "Export Hermes events to Swift",
+	Long: `Export Hermes events to Swift storage container.
+Exports can be saved in different formats (json, csv, yaml) for further processing or archival.`,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if err := viper.BindPFlags(cmd.Flags()); err != nil {
+			return fmt.Errorf("failed to bind flags: %w", err)
+		}
+
+		// Validate container name is provided
+		if viper.GetString("container") == "" {
+			return errors.New("container name is required")
+		}
+
+		// Validate format
+		_, err := ParseExportFormat(viper.GetString("format"))
+		if err != nil {
+			return err
+		}
+
+		return verifyGlobalFlags(defaultListKeyOrder)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+
+		// Warn if trying to export more than default limit
+		if viper.GetInt("limit") > maxOffset {
+			fmt.Fprintf(os.Stderr, "Warning: Exporting more than %d events may take a long time.\n\n", maxOffset)
+		}
+
+		// Get events using existing list functionality
+		client, err := NewHermesV1Client(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to create Hermes client: %w", err)
+		}
+
+		// Configure debug logging if enabled
+		if viper.GetBool("debug") {
+			transport := &http.Transport{}
+			client.HTTPClient = http.Client{
+				Transport: transport,
+			}
+		}
+
+		fmt.Fprintf(os.Stderr, "Fetching events...\n")
+
+		var allEvents []events.Event
+		var bar *pb.ProgressBar
+
+		logg.Debug("fetching events matching specified criteria")
+
+		listOpts := buildListOpts()
+		if err = getEvents(ctx, client, &allEvents, listOpts, viper.GetInt("limit"), true, &bar); err != nil {
+			if bar != nil {
+				bar.Finish()
+			}
+			return fmt.Errorf("failed to list events: %w", err)
+		}
+		if bar != nil {
+			bar.Finish()
+		}
+
+		if len(allEvents) == 0 {
+			return errors.New("no events found matching the specified criteria")
+		}
+
+		fmt.Fprintf(os.Stderr, "\nFound %d events to export\n", len(allEvents))
+
+		// Convert events to desired format
+		fmt.Fprintf(os.Stderr, "Converting to %s format...\n", viper.GetString("format"))
+		var buf bytes.Buffer
+		if err := convertToRequestedFormat(&buf, allEvents, viper.GetString("format")); err != nil {
```

Comment:
previously, on the 139 line the err is used as already defined.
```suggestion
		if err = convertToRequestedFormat(&buf, allEvents, viper.GetString("format")); err != nil {
```